pkg/mkverification-raw-efi/make-raw

```diff
@@ -0,0 +1,457 @@
+#!/bin/sh
```

Comment:
It looks like you just copied make-raw from mkimage-raw-efi package. This duplication creates two code bases that shall be maintained and kept in sync... have you tried to share the same script across the two packages?

Reply:
I understand the comment but I am not sure how to do it, this is the entry point of the Dockerfile of mkverification-raw-efi. Should I make a soft link? Is this a good idea?

Reply:
@jsfakian for other cases (such as telling kvm and xen images apart) we build a single image and then a boot/runtime we check the name of the image.  I don't know if the same approach makes sense here.

In such a case you could have a new Linuxkit onboot container run the verification, and it would check the name being e.g., *verification-amd64.

But I guess you need the ability to add some test VM images in the EVE image as well. I wonder if it would be more maintainable to take the output from mkimage-raw-efi and add additional pieces to it.

Reply:
It takes work to do so. I will think about how to do it in future iterations.

Reply:
If they are identical then a soft link would make sense. I think we have cases where softlinks are checked into github already.